core/src/main/java/org/apache/carbondata/core/cache/CarbonLRUCache.java

```diff
@@ -327,15 +309,15 @@ public Cacheable get(String key) {
    */
   public void clear() {
     synchronized (lruCacheMap) {
-      for (Cacheable cachebleObj : lruCacheMap.values()) {
+      for (Cacheable cachebleObj : lruCacheMap.asMap().values()) {
```

Comment:
why not use method `invalidateAll()` ?

Reply:
the above code is for invalidating Cacheable objects. For LRUCache, cleanUp method is used for clearing the cache.